go/engine/revoke.go

```diff
@@ -105,18 +108,22 @@ func (e *RevokeEngine) Run(ctx *Context) error {
 	e.G().LocalSigchainGuard().Set(ctx.GetNetContext(), "RevokeEngine")
 	defer e.G().LocalSigchainGuard().Clear(ctx.GetNetContext(), "RevokeEngine")
 
+	me, err := libkb.LoadMe(libkb.NewLoadUserArg(e.G()))
+	if err != nil {
+		return err
+	}
+
 	currentDevice := e.G().Env.GetDeviceID()
 	var deviceID keybase1.DeviceID
 	if e.mode == RevokeDevice {
 		deviceID = e.deviceID
 		if e.deviceID == currentDevice && !e.force {
 			return errors.New("Can't revoke the current device.")
 		}
-	}
 
-	me, err := libkb.LoadMe(libkb.NewLoadUserArg(e.G()))
-	if err != nil {
-		return err
+		if len(me.GetComputedKeyFamily().GetAllActiveDevices()) == 1 && !e.forceLast {
+			return errors.New("Can't revoke the last device.")
+		}
```

Comment:
Will this error bubble all the way up?  We probably want to tell users that can override with a flag, right? But then this isn't the right place for it, so we should probably have an exportable error that we can format on the front end?  What do you think?